src/utils.rs

```diff
@@ -141,8 +142,54 @@ impl Console {
     }
 
     /// Read a chunk of raw data from the console and output it.
-    pub fn read_to(&self, output: &mut dyn Write) -> NitroCliResult<()> {
+    pub fn read_to(
+        &self,
+        output: &mut dyn Write,
+        disconnect_timeout_sec: Option<u64>,
+    ) -> NitroCliResult<()> {
+        let epoll_fd = epoll::epoll_create().map_err(|e| {
```

Comment:
The epoll logic should run only if ```disconnect_timeout_sec``` is not None. Otherwise we affect the performance of the ```read_to()``` function when it's called without a disconnect interval.